<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="compressButton">
        <property name="text">
         <string>Compress Space</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
   </widget>
   <widget class="QMenu" name="menuComplex_Convertion">
    <property name="title">
     <string>&amp;Complex Convertion</string>
    </property>
   </widget>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="openChainAction"/>
    <addaction name="saveChainAction"/>
    <addaction name="separator"/>
    <addaction name="batchConvertionAction"/>
   </widget>
   <widget class="QMenu" name="windowMenu">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuComplex_Convertion"/>
   <addaction name="windowMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Simple tools</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="colorDockWidget">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Color</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="ColorChooser" name="colorWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="statusTip">
    <string>Undo</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="statusTip">
    <string>Redo</string>
   </property>
  </action>
  <action name="To_black_and_whiteaction">
   <property name="text">
    <string>To black and white</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open an image</string>
   </property>
   <property name="statusTip">
    <string>Open an image</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save As</string>
   </property>
   <property name="toolTip">
    <string>Save an image</string>
   </property>
   <property name="statusTip">
    <string>Save an image</string>
   </property>
  </action>
  <action name="saveChainAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/save_chain.png</normaloff>:/images/save_chain.png</iconset>
   </property>
   <property name="text">
    <string>Save Process Chain</string>
   </property>
   <property name="toolTip">
    <string>Save process chain</string>
   </property>
   <property name="statusTip">
    <string>Save process chain</string>
   </property>
  </action>
  <action name="openChainAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/open_chain.png</normaloff>:/images/open_chain.png</iconset>
   </property>
   <property name="text">
    <string>Open Process Chain</string>
   </property>
   <property name="iconText">
    <string>Open process chain</string>
   </property>
   <property name="toolTip">
    <string>Open process chain</string>
   </property>
   <property name="statusTip">
    <string>Open process chain</string>
   </property>
  </action>
  <action name="batchConvertionAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/batch_convertion.png</normaloff>:/images/batch_convertion.png</iconset>
   </property>
   <property name="text">
    <string>Batch Convertion</string>
   </property>
   <property name="iconText">
    <string>Batch convertion</string>
   </property>
   <property name="toolTip">
    <string>Batch convertion</string>
   </property>
  </action>
  <action name="newAction">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create an image</string>
   </property>
   <property name="statusTip">
    <string>Create an image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorChooser</class>
   <extends>QWidget</extends>
   <header>colorchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
